packages/core/src/actors/transition.ts

```diff
@@ -2,14 +2,16 @@ import { ActorBehavior, ActorContext, EventObject } from '../types';
 import { isSCXMLEvent } from '../utils';
 
 /**
- * Returns an actor behavior from a reducer and its initial state.
+ * Returns an actor behavior from a transition function and its initial state.
  *
- * @param transition The pure reducer that returns the next state given the current state and event.
- * @param initialState The initial state of the reducer.
+ * A transition function is a function that takes the current state and an event and returns the next state.
+ *
+ * @param transition The pure transition function that returns the next state given the current state and event.
```

Comment:
i would refrain from calling this a "pure" function - since it can utilize `actorContext` to execute side-effects